app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryUpdate.kt

```diff
@@ -45,14 +45,21 @@ class CustomDrugEntryUpdate : Update<CustomDrugEntryModel, CustomDrugEntryEvent,
       model: CustomDrugEntryModel,
       drug: Drug
   ): Next<CustomDrugEntryModel, CustomDrugEntryEffect> {
+    val cursorPosition = if (drug.dosage != null) cursorPositionFromDosage(drug.dosage) else 0
```

Comment:
Do we need the to return 0 twice here? `cursorPositionFromDosage` is going to return us 0, can we instead check the nullability inside the function and have one condition (either x position or 0 position. WDYT?